src/Plugin/views/argument_default/Membership.php

```diff
@@ -0,0 +1,149 @@
+ *
+ * @ViewsArgumentDefault(
+ *   id = "og_group_membership",
+ *   title = @Translation("Group memberships from current user")
+ * )
+ */
+class Membership extends ArgumentDefaultPluginBase implements CacheableDependencyInterface {
+
+  /**
+   * The OG context provider.
+   *
+   * @var \Drupal\Core\Plugin\Context\ContextProviderInterface
+   */
+  protected $ogContext;
+
+  /**
+   * The OG membership manager.
+   *
+   * @var \Drupal\og\MembershipManagerInterface
+   */
+  protected $ogMembership;
+
+  /**
+   * The user to be evaluated.
+   *
+   * @var \Drupal\Core\Session\AccountInterface
+   */
+  protected $ogUser;
+
+  /**
+   * Constructs a new User instance.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param mixed $plugin_definition
+   *   The plugin implementation definition.
+   * @param \Drupal\Core\Plugin\Context\ContextProviderInterface $og_context
+   *   The OG context provider.
+   * @param \Drupal\og\MembershipManagerInterface $og_membership
+   *   The OG membership manager.
+   * @param \Drupal\Core\Session\AccountInterface $og_user
+   *   The user to be evaluated.
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, ContextProviderInterface $og_context, MembershipManagerInterface $og_membership, AccountInterface $og_user) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+
+    $this->ogContext = $og_context;
+    $this->ogMembership = $og_membership;
+    $this->ogUser = $og_user;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('og.context'),
+      $container->get('og.membership_manager'),
+      $container->get('current_user')->getAccount()
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function defineOptions() {
+    $options = parent::defineOptions();
+    $options['og_group_membership'] = ['default' => ''];
+
+    return $options;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildOptionsForm(&$form, FormStateInterface $form_state) {
+    $form['og_group_membership'] = [];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getArgument() {
+    // Currently restricted to node entities.
+    return implode(',', $this->getCurrentUserGroupIds('node'));
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheMaxAge() {
+    return Cache::PERMANENT;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheContexts() {
+    return ['og_role'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCacheTags() {
+    $group = $this->getGroup();
```

Comment:
The `getGroup()` methods was removed, so this will fail.

```
-  protected function getGroup() {
-    $contexts = $this->ogContext->getRuntimeContexts(['og']);
-    if (!empty($contexts['og']) && $group = $contexts['og']->getContextValue()) {
-      if ($group instanceof ContentEntityInterface) {
-        return $group;
-      }
-    }
-  }
```

Reply:
@tombola this now affects our project. Please update?

Reply:
@zerolab, is this still valid?

Reply:
I added it in f6e0e52, so sorted